README.md

```diff
@@ -114,6 +115,8 @@ bundle exec ruby lib/get-deleted-user-ids-to_local-file.rb
 ```
 bundle exec ruby lib/purge-users-using-local-file.rb | tee data/`date --iso-8601='date'`.log
 ```
+bundle exec ruby lib/count-closed-tickets-by-year.rb
+bundle exec ruby lib/get-latest-ticket-numbers.rb
```

Comment:
Can we move the last two commands to line 117 before triple backticks?

Reply:
Yes, agreed but I will handle in a separate PR because I have another change to add to the README.